pages/index.js

```diff
@@ -1,3 +1,15 @@
+import PlantCard from "@/components/PlantCard";
+import { createGlobalStyle } from "styled-components";
+import styled from "styled-components";
+import TitleBar from "@/components/TitleBar";
+
 export default function HomePage() {
-  return <h1>Hello from Next.js</h1>;
+  return (
+    <>
+      <main>
+        <TitleBar />
+        <PlantCard />
+      </main>
+    </>
+  );
 }
```

Comment:
You can think about moving the header to the _app later on, I suppose you want to have it on each side. 
The structure of the body is: 
header
main
footer (if you have one)

Reply:
In your US you write "the title is always visible".
That is not the case: 
You can fix this with:  
position: fixed; 
the parent element needs position: relative;

Reply:
It's a really good start! 
Most of the things I commented are only small things.
Ask me if you have further questions:-)